### PR TITLE
DOCSP-27817 fix facet type to facet name per DOP

### PR DESCRIPTION
**Pull request by sarahemlin**
Hey @caitlindavey & @mongoKart,

I found out from DOP that the facet syntax should be :name: instead of :type:, so here are fixes to the affected docs-ecosystem files.

Thank you!
Sarah

JIRA - <https://jira.mongodb.org/browse/DOCSP-27817>

### CODE REVIEW

File: source/java-drivers.txt [hunk 1]
```diff
@@ -7,11 +7,11 @@ MongoDB Java Drivers
 .. default-domain:: mongodb
 
 .. facet::
-  :type: programming_language
+  :name: programming_language
```
Comment (ccho-mongodb) [inline source/java-drivers.txt:10]: Suggestion:
I think these should be indented one more space, similar to the facet attributes in the pymongo.txt file in this PR.

**Review by ccho-mongodb — APPROVED** [review]
LGTM, but I think it would be good to address the indentation in source/java-drivers.txt before merging.

File: source/java-drivers.txt (hunk 1 above)
Comment (ccho-mongodb) [inline source/java-drivers.txt:10]:
```suggestion
   :name: programming_language
```

File: source/java-drivers.txt [hunk 2]
```diff
@@ -7,11 +7,11 @@ MongoDB Java Drivers
...
   :values: java
 
 .. facet::
-  :type: genre
+  :name: genre
```
Comment (ccho-mongodb) [inline source/java-drivers.txt:14]:
```suggestion
   :name: genre
```

**Review by sarahemlin — COMMENTED** [review]
@ccho-mongodb I think this one is ready to merge now.  Thank you!

**Comment by ccho-mongodb** [comment]
Hi @sarahemlin. I've made the same changes already in another merged PR, so closing this one!

**Pull request closed without merging**